plugin/storage/es/setup.cfg

```diff
@@ -0,0 +1,3 @@
+[metadata]
+name = jaeger-es
+version = 1.25.0
```

Comment:
How important is it to have an accurate version number here? Feels like a serious overhead to the process to update this very minor corner.

Reply:
I believe this is relevant to consumers of this package, if this package was to be published at some central repository, which isn't happening for these scripts.

Reply:
Our python scripts are not published anywhere. Can we omit this file?

Reply:
I'm not a Python expert, but apparently, some tools do expect Python projects to contain at least a setup.cfg and the requirements.txt files. Perhaps we can have 0.0.1 as the version?

Reply:
the question is which tool requires setup.cfg in jaeger. Is it https://github.com/jazzband/pip-tools?

Reply:
In my specific case, it's Cachito, but I assume this might be true for other similar tools: https://github.com/release-engineering/cachito/blob/master/docs/pip.md#setuppy-setupcfg